protocol-designer/src/pages/CreateNewProtocolWizard/SelectPipettes.tsx

```diff
@@ -89,9 +91,12 @@ export function SelectPipettes(props: WizardTileProps): JSX.Element | null {
   //  initialize pipette name once all fields are filled out
   React.useEffect(() => {
     if (pipetteType != null && pipetteVolume != null) {
-      setValue(`pipettesByMount.${defaultMount}.pipetteName`, selectedPip)
+      setValue(
+        `pipettesByMount.${defaultMount}.pipetteName`,
+        selectedPipetteName
+      )
     }
-  }, [pipetteType, pipetteGen, pipetteVolume, selectedPip])
+  }, [pipetteType, pipetteVolume, selectedPipetteName])
```

Comment:
`pipetteGen` is needed for an ot-2 pipette. Maybe we need to do something like:

```suggestion
 if (pipetteType != null && pipetteVolume != null && (robotType === OT2_ROBOT_TYPE && pipetteGen != null)) {
      setValue(
        `pipettesByMount.${defaultMount}.pipetteName`,
        selectedPipetteName
      )
    }
  }, [pipetteType, pipetteGen, pipetteVolume, selectedPipetteName])
```